pkg/test/server.go

```diff
@@ -43,9 +42,9 @@ const (
 type Hasher struct {
 }
 
-// Hash function that always returns same value.
-func (h Hasher) Hash(*core.Node) (cache.Key, error) {
-	return cache.Key(node), nil
+// ID function that always returns the same value.
+func (h Hasher) ID(*core.Node) string {
```

Comment:
NodeId

Reply:
Is there something else that needs to be IDed except nodes?